src/index_intf.ml

```diff
@@ -128,6 +128,7 @@ module type S = sig
     ?cache:cache ->
     ?fresh:bool ->
     ?readonly:bool ->
+    ?throttle:[ `Overcommit_memory | `Block_writes ] ->
```

Comment:
Let's add an explanatory line in the doc-string (even though I think the type is very self-explanatory). At least, it makes sense to say what the default is.